pkg/gui/presentation/files.go

```diff
@@ -165,6 +168,10 @@ func getFileLine(
 		output += theme.DefaultTextColor.Sprint(" (submodule)")
 	}
 
+	if lineChanges := formatLineChanges(file); showNumberOfLineChanges && lineChanges != "" {
```

Comment:
I'd wrap this whole thing in `if showNumberOfLineChanges` rather than first computing `formatLineChanges`: generally the earlier you can short-circuit, the better